README.md

```diff
@@ -1,5 +1,7 @@
 # Informo specifications
 
+[![https://specs.informo.network](https://img.shields.io/badge/specs-specs.informo.network-brightgreen.svg?style=flat-square)](https://specs.informo.network) [![FDL-1.3](https://img.shields.io/badge/license-FDL--1.3-blue.svg?style=flat-square)](/LICENSE) [![Open proposals](https://img.shields.io/badge/dynamic/json.svg?label=Open%20proposals&url=https%3A%2F%2Fstats.informo.network%2Fspecs%2Fproposals%3Fstate%3Dopen&query=%24.length&style=flat-square)](https://github.com/Informo/specs/pulls?q=is%3Apr+is%3Aopen) [![Merged proposals](https://img.shields.io/badge/dynamic/json.svg?label=Merged%20proposals&url=https%3A%2F%2Fstats.informo.network%2Fspecs%2Fproposals%3Fstate%3Dmerged&query=%24.length&&style=flat-square)](https://github.com/Informo/specs/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed) [![#discuss:weu.informo.network on Matrix](https://img.shields.io/matrix/discuss:weu.informo.network.svg?server_fqdn=fr-02.nodes.informo.network&logo=matrix&style=flat-square)](https://matrix.to/#/#discuss:weu.informo.network)
```

Comment:
First shield: `spec | spec.informo.network`, maybe `View | spec.informo.network` or `Online | spec.informo.network` to avoid double "spec" word

Reply:
Not sure. I had troubles figuring out what to say there tbh. Maybe "live"?